docs/developer-experience/readme.md

```diff
@@ -0,0 +1,231 @@
+# bauta.js and developer experience
+
+## Audience of this document
+
+This document is for anyone that is starting to learn about bauta.js library. For example a developer that has just discovered bauta.js or a techical lead in a team that is wondering whether bauta.js could be helpful to them or cover their needs.
+
+## Reason for this document
+
+The goal of this document is to help you to answer the question: "Do I want to use bauta.js?" and provide you a starting point answering the question "How do i get started with bauta.js?".
+
+We hope that by reading this document, that should not take more than 15 or 20 minutes, you can review quickly what is bauta.js about and let you determine if it could be helpful to you.
```

Comment:
```suggestion
We hope that by reading this document, which should take at most 15 or 20 minutes, you can quickly review what bauta.js is about and let you determine if it could be helpful to you.
```